### DISCUSSION TITLE
How to modify `og:title`?

### DISCUSSION THREAD

**Discussion by xlionjuan** [nunocoracao/blowfish#1385] [Q&A]
> I've post same question on Hugo: https://discourse.gohugo.io/t/how-to-add-website-title-on-posts-og-title/49118

EX:
Website Title: My Awesome Website
Article Title: My first post

It will generate `<meta property="og:title" content="My first post">`
But I want it be: `<meta property="og:title" content="My first post - My Awesome Website">`
What should I do?

I found related config on GitHub, but it seens a part of Hugo, I don’t know how to modify.

https://github.com/gohugoio/hugo/blob/f0a26cf58e9bfbe55c091d51a4133bc22cad5d1f/tpl/tplimpl/embedded/templates/opengraph.html#L1

<hr>

[jmooring](https://discourse.gohugo.io/u/jmooring) replied me said I should copy `opengraph.html` from Hugo's template to  `layouts/partials`, and call it from your template using the [partial](https://gohugo.io/functions/partials/include/) function:

`{{ partial "opengraph.html" . }}`

If I'm not wrong, I should copy something from `themes/blowfish/layouts` to `layouts/` than modify it, but I don't know which file is related to my question.

**Comment by nunocoracao** [comment]
@xlionjuan hey, check which files are generating the OG tags. I think they should be in the default folder. You can then override what goes into each tag.
